src/anm/analytics.js

```diff
@@ -3,15 +3,26 @@ var engine = require('engine'),
 
 var Analytics = function () {
     var self = this,
-        animatronUrl = utils.makeApiUrl('/analytics/player');
+        animatronUrl = utils.makeApiUrl('analytics', '/analytics/player');
 
     self.enabled = animatronUrl != null && animatronUrl.indexOf('animatron-test') >= 0;
-    self.queue = [];
+    self.queue = {};
 
-    var event = function () {
-        if (self.queue.length > 0) {
-            var data = JSON.stringify(self.queue);
-            self.queue = [];
+    var event = function (e) {
+        if (e && e.type === 'unload') {
+            anm.player_manager.instances.forEach(function (player) {
+                if (player && player.state && player.canvas) {
+                    self.trackPlayingComplete(player);
+                }
+            })
+        }
+        if (utils.is.not_empty(self.queue)) {
```

Comment:
To make server know analytics were initialized, but no tracking events happened. Probably.